packages/web-app-files/src/mixins/actions/restore.ts

```diff
@@ -120,6 +247,42 @@ export default {
         const user = await this.$client.users.getUser(this.user.id)
         this.SET_QUOTA(user.quota)
       }
+    },
+    async $_restore_trigger({ resources }) {
+      //! collect and request existing files in associated parent folders of each resource
```

Comment:
Why the `//!`? Just curious, as I don't know this in JS/TS context... would prefer `//` without the `!` if this is not some convention I simply don't know about. Please enlighten me. :-) (Same for the `//?` above)